bulbs/poll/api.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.routers import DefaultRouter
+
+from .viewsets import PollViewSet, AnswerViewSet
+
+router = DefaultRouter(trailing_slash=True)
+
+api_v1_router = DefaultRouter()
+api_v1_router.register(
```

Comment:
@mparent61 I think we use flake8, not sure how the plugin might work for vim, but :ahuh: